core/tests/puppeteer-acceptance-tests/specs/logged-out-user/click-all-buttons-on-teach-page.spec.ts

```diff
@@ -55,6 +55,19 @@ describe('Logged-out User in Teach page', function () {
     DEFAULT_SPEC_TIMEOUT_MSECS
   );
 
+  it(
+    'should open the blog page with teacher stories when "Check out our blog" button is clicked',
+    async function () {
+      await loggedOutUser.clickBlogButtonInTeachPage();
+      const currentUrl = await loggedOutUser.getCurrentUrl();
+      expect(currentUrl).toContain(
+        '/blog/search/find?q=&tags=(%22Teacher%20story%22)'
+      );
+      expect(currentUrl).not.toContain('oppia.org');
```

Comment:
I think it's fine to not include this assertion. It's weird to do it for this page and not all the other pages that are internal website links.

I think you can just drop this line.